src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo/CommentInfo';
```

Comment:
it's good practice to create an index file inside the folder, to allow write imports shorter
```suggestion
import { CommentInfo } from '../CommentInfo';
```